file_reader_code.R

```diff
@@ -0,0 +1,66 @@
+if (!require("pacman")) install.packages("pacman")
+library(pacman)
+pacman::p_load(dplyr, readxl, data.table, tidyr)
```

Comment:
Why have you used pacman here rather than standard library calls? Have you looked at something like renv for a standard approach to package management?